packages/webapp/webapp.d.ts

```diff
@@ -1,5 +1,5 @@
 import * as http from 'http';
-import * as connect from 'connect';
+import e from '@types/express';
```

Comment:
You shuold never import `@types/*` directly:
```suggestion
import express from 'express';
```